### PR TITLE
Markdown conversion for `ru` - Replace - MDN section :warning: Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of #8539
Conversion report: https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-29t10_41_31-812z-md

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/ru/mdn/contribute/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  - Гайд
+  - Landing
+  - MDN Meta
+translation_of: MDN/Contribute
+---
+{{MDNSidebar}}
+
+MDN Web Docs нужна ваша помощь! У нас есть много опечаток и ошибок для исправления, примеров для написания, людей, с которыми можно поговорить и много чего другого. Количество всего этого растёт вместе с количеством людей, использующих сайт. На этой странице рассказывается, что вы можете сделать, чтобы помочь.
+
+> **Примечание:** #### ПримечаниеЕсли вы никогда до этого не вносили вклад в MDN, статья [начало работы](/ru/docs/MDN/Contribute/Getting_started) объяснит процесс в четырёх простых шагах. Если вы читаете это сообщение, для вас есть хорошая новость — вы уже на шаге 3: "Узнайте, как вы можете помочь"!
```
Comment (sashasushko) [inline files/ru/mdn/contribute/index.md:14]:
```suggestion
> **Примечание:** Если вы никогда до этого не вносили вклад в MDN, статья [начало работы](/ru/docs/MDN/Contribute/Getting_started) объяснит процесс в четырёх простых шагах. Если вы читаете это сообщение, для вас есть хорошая новость — вы уже на шаге 3: "Узнайте, как вы можете помочь"!
```

File: files/ru/mdn/contribute/index.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+          </li>
+          <li>
+            Достаточное владение английским языком: он действительно не должен
+            быть идеальным.
+          </li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
```
Comment (sashasushko) [inline files/ru/mdn/contribute/index.md:126]:
```suggestion
<table class="standard-table">
    <thead>
        <tr>
            <th scope="col">Задачи</th>
            <th scope="col">Описание</th>
            <th scope="col">Необходимые навыки</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td><a href="/en-US/docs/MDN/Contribute/Fixing_MDN_content_bugs">Исправление ошибок в контенте MDN</a></td>
            <td>Это наш <a href="https://github.com/mdn/content/issues">репозиторий контента</a>, в котором люди сообщают о проблемах найденных в документации MDN (вы так же можете найти ошибки в старом <a href="https://github.com/mdn/sprints/">репозитории sprints</a>, но в конечном счёте мы его закроем). Мы получаем очень много ошибок, связанных с контентом, и мы будем признательны за любую помощь, которую вы можете оказать в их исправлении.</td>
            <td>
                <ul>
                    <li>Знание веб-технологий, с которыми вы хотите помочь (например, JavaScript, CSS).</li>
                    <li>Приемлемое знание английского языка (не беспокойтесь, если ваш английской не идеален: мы можем помочь с этим).</li>
                </ul>
            </td>
        </tr>
        <tr>
            <td><a href="https://github.com/mdn/content/blob/main/REVIEWING.md">Ревью правок MDN</a></td>
            <td>Люди постоянно отправляют пулреквесты в наш <a href="https://github.com/mdn/content">репозиторий контента</a> для обновления MDN, и нам нужна помощь в их ревью. Посмотрите страницу <a href="https://github.com/mdn/content/blob/main/REVIEWING.md">REVIEWING.md</a>, чтобы узнать как проходит процесс ревью и какую помощь вы можете оказать.</td>
            <td>
                <ul>
                    <li>Знание веб-технологий, с которыми вы хотите помочь (например, JavaScript, CSS).</li>
                    <li>Приемлемое знание английского языка (не беспокойтесь, если ваш английской не идеален: мы можем помочь с этим).</li>
                </ul>
            </td>
        </tr>
        <tr>
            <td><a href="/en-US/docs/MDN/Contribute/Help_beginners">Помощь новичкам в учёбе на MDN</a></td>
            <td>Наши страницы <a href="/ru/docs/Learn">«Изучение веб-разработки»</a> просматриваются более миллиона раз в месяц и имеют <a href="https://discourse.mozilla.org/c/mdn/learn/250">активные форумы</a>, куда люди обращаются за помощью или просят оценить их работу. Мы рады помощи с ответами на задаваемые вопросы и помощи в развитии нашего сообщества учеников.</td>
            <td>
                <ul>
                    <li>Знание веб-технологий, с которыми вы хотите помочь (например, JavaScript, CSS).</li>
                    <li>Энтузиазм в объяснении технических тем и помощи новичками в изучении программирования.</li>
                    <li>Достаточное владение английским языком: он действительно не должен быть идеальным.</li>
                </ul>
            </td>
        </tr>
    </tbody>
</table>
```

File: files/ru/mdn/tools/kumascript/troubleshooting/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+## TemplateLoadingError
+
+`TemplateLoadingError` ошибка появляется когда KumaScript не может найти [макрос](/ru/docs/MDN/Contribute/Structures/Macros) для добавления на страницу.
+
+Необходимо проверить:
+
+- Опечатка в имени макроса или он был переименован.
+  - : Вы можете взглянуть на список известных макросов в [GitHub репозитории](https://github.com/mdn/kumascript/tree/master/macros).
+
+> **Примечание:** **Совет:** вы можете легко и быстро перейти к определённому макросу, используя [поиск по ключевым словам](http://kb.mozillazine.org/Using_keyword_searches) в Firefox. Смотрите {{SectionOnPage("/ru/docs/MDN/Contribute/Tools/KumaScript", "Использование поиска по ключевым словам для открытия шаблона страницы")}} для пошагового справочника по созданию поиска по ключевым словам.
```
Comment (sashasushko) [inline files/ru/mdn/tools/kumascript/troubleshooting/index.md:43]:
```suggestion
> **Совет:** вы можете легко и быстро перейти к определённому макросу, используя [поиск по ключевым словам](http://kb.mozillazine.org/Using_keyword_searches) в Firefox. Смотрите {{SectionOnPage("/ru/docs/MDN/Contribute/Tools/KumaScript", "Использование поиска по ключевым словам для открытия шаблона страницы")}} для пошагового справочника по созданию поиска по ключевым словам.
```

File: files/ru/mdn/tools/unsupported_get_api/index.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+  - : Указывает Kuma вернуть содержимое только из раздела с указанным якорем/именем привязки.
+
+    **Пример:**
+
+    - [/ru/docs/MDN/Tools/URL-suffix?raw\&section=params](/ru/docs/MDN/Tools/URL-suffix?raw&section=params)
+      (...и больше интересного...)
+    - [/ru/docs/MDN/Tools/URL-suffix?raw\&macros\&section=params](/ru/docs/MDN/Tools/URL-suffix?raw&macros&section=params)
+
+    > **Примечание:** **Уведомление об ошибке:** В настоящее время существует ошибка, из-за которой параметр `section` возвращает весь документ, если вы также не укажете параметр `raw`.
```
Comment (sashasushko) [inline files/ru/mdn/tools/unsupported_get_api/index.md:64]:
```suggestion
    > **Уведомление об ошибке:** В настоящее время существует ошибка, из-за которой параметр `section` возвращает весь документ, если вы также не укажете параметр `raw`.
```

File: files/ru/mdn/writing_guidelines/howto/write_a_new_entry_in_the_glossary/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+### Шаг 1: Выберите термин, которому хотите дать определение
+
+Первое, что нужно сделать, это выбрать термин, о котором писать. Если вы не знаете, какие термины нуждаются в определениях, вы можете [проверить наш список предложений](/ru/docs/Glossary#Contribute_to_the_glossary). Просто нажмите на термин и начните писать. Если вы уже авторизованы в системе, то вы попадёте в Редактор MDN.
+
+### Шаг 2: Дайте краткое определение
+
+Первый абзац любой страницы глоссария – это простое и краткое определение/описание термина (желательно не длиннее двух предложений). Удостоверьтесь, что любой, кто читает определение, сможет сразу понять определяемый термин.
+
+> **Примечание:** **Замечание:** Пожалуйста, не копируйте определения из других источников (особенно из Википедии, поскольку её спектр лицензионных версий меньше, и, таким образом, несопоставим с MDN). Ваша запись должна быть уникальной, а не скопированной.
```
Comment (sashasushko) [inline files/ru/mdn/writing_guidelines/howto/write_a_new_entry_in_the_glossary/index.md:38]:
```suggestion
> **Примечание:** Пожалуйста, не копируйте определения из других источников (особенно из Википедии, поскольку её спектр лицензионных версий меньше, и, таким образом, несопоставим с MDN). Ваша запись должна быть уникальной, а не скопированной.
```

File: files/ru/mdn/writing_guidelines/howto/write_a_new_entry_in_the_glossary/index.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+#### Отредактируйте всплывающие подсказки
+
+Как и в случае с большинством ссылок MDN, наведение курсора мыши на ссылку глоссария покажет краткое описание страницы (в данном случае краткое описание определения термина). По умолчанию подсказка представляет собой весь первый абзац статьи, но в большинстве случаев такое описание может оказаться слишком большим.
+
+Вы можете сами указать фрагмент текста, который в дальнейшем будет использоваться как текст всплывающей подсказки при ссылке на вашу запись. Выберите одно или два предложения в записи, которые лучше всего описывают термин, а затем добавьте к ним стиль "SEO Summary" (резюме). Это действие не только создаст текст подсказки, но и определит текст, который будет отправлен в Google и другие поисковые системы, когда они запросят краткое резюме, описывающее содержимое страницы. Таким образом, вашу запись будет легче находить с помощью поисковиков.
+
+> **Примечание:** Примечание: В идеале, резюме должно содержать около 150-160 символов. Лучше меньше, да лучше.
```
Comment (sashasushko) [inline files/ru/mdn/writing_guidelines/howto/write_a_new_entry_in_the_glossary/index.md:56]:
```suggestion
> **Примечание:** В идеале, резюме должно содержать около 150-160 символов. Лучше меньше, да лучше.
```

File: files/ru/mdn/writing_guidelines/howto/write_a_new_entry_in_the_glossary/index.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+### Шаг 3: Добавьте ссылки
+
+Заканчиваться запись в глоссарии всегда должна разделом "Узнать больше". Этот раздел должен содержать ссылки, которые помогут читателю подробнее изучить вопрос: узнать больше деталей, научиться использовать соответствующие технологии и так далее.
+
+Мы рекомендуем сортировать ссылки по трём группам:
+
+_**Общие знания**_
+Ссылки, которые предоставляют информацию более общего характера, например, ссылка на[ Википедию ](http://wikipedia.org/)– это хорошая отправная точка.
```
Comment (sashasushko) [inline files/ru/mdn/writing_guidelines/howto/write_a_new_entry_in_the_glossary/index.md:65]:
```suggestion
Ссылки, которые предоставляют информацию более общего характера, например, ссылка на [Википедию](https://wikipedia.org/) – это хорошая отправная точка.
```

**Pull request merged**